src/neptune_scale/core/components/aggregating_queue.py

```diff
@@ -147,13 +145,37 @@ def get(self) -> BatchedOperations:
             time.process_time() - start,
         )
 
+        batch = create_run_batch(batch_operations)
+
         return BatchedOperations(
             sequence_id=batch_sequence_id,
             timestamp=batch_timestamp,
             operation=batch.SerializeToString(),
         )
 
 
+def create_run_batch(operations: list[(float, RunOperation)]) -> RunOperation:
+    if len(operations) == 1:
+        return operations[0][1]
+
+    batch = RunOperation()
+
+    head_operation = operations[0][1]
+    batch.project = head_operation.project
+    batch.run_id = head_operation.run_id
+    batch.create_missing_project = head_operation.create_missing_project
+    batch.api_key = head_operation.api_key
+
+    for _, operation in operations:
+        operation_type = operation.WhichOneof("operation")
+        if operation_type == "update":
+            batch.update_batch.snapshots.append(operation.update)
+        else:
+            logger.error("Cannot batch operation of type %s", operation_type)
```

Comment:
I think we should error out here, there shouldn't be any other types of updates. Unless at some point we decide that batching run creation is fine